WATOLINK/bci-gaming
# Alexis neurosity

## BCI-integration
[Integrate Neurosity and Blink detection]



## Implementation description
Integrated the Unity-Neurosity sdk into our repo
Added a script for blink detection in the Blink detection folder in scripts 
Modified that RawBrainwaves Handler to call our blink detection and print BLINK DETECTED when the conditions are met



## Steps to test
1. Make sure you have git LFS set up 
2. Install the dependencies needed in the README 
3. Open the game and make sure there are no errors in the console that prevent you from running



## What should reviewers focus on?
Will make a new ticket to integrate in the actual game once an integration scene is created


## Checklist
